diff --git a/node_modules/@pantheon-systems/pds-toolkit-react/_dist/index.js b/node_modules/@pantheon-systems/pds-toolkit-react/_dist/index.js
index 347b73b..3c2f50e 100644
--- a/node_modules/@pantheon-systems/pds-toolkit-react/_dist/index.js
+++ b/node_modules/@pantheon-systems/pds-toolkit-react/_dist/index.js
@@ -111,6 +111,12 @@ const h2 = {
   resolveCrumb: () => {
   }
 }, N1 = Ct(h2), Ue = () => Math.random().toString(32).substring(2, 13), k1 = (t, s) => {
+
+  // Check for children first.
+	if (!node?.childNodes) {
+		return;
+	}
+
   let n;
   for (s = s || [], n = 0; n < t.childNodes.length; n++)
     s.push(t.childNodes[n]), k1(t.childNodes[n], s);
@@ -3398,8 +3404,9 @@ const P2 = m.exact({
   children: t,
   mobileMenuMaxWidth: s = 640
 }) => {
+  const isBrowser = typeof window !== 'undefined';
   const [n, a] = F(
-    window.innerWidth !== void 0 ? window.innerWidth : 1024
+    isBrowser && window.innerWidth !== void 0 ? window.innerWidth : 1024
   );
   re(() => (a(window.innerWidth), window.addEventListener("resize", c), () => {
     window.removeEventListener("resize", c);
